serve/benchmarks/benchmark_throughput.py

```diff
@@ -142,13 +144,14 @@ def main(args: argparse.Namespace):
     elapsed_time = run_mlc(
         requests,
         engine,
+        args.num_sequences_to_sample,
     )
 
     if args.use_staging_engine:
         engine.stop()
 
     total_num_tokens = sum(
-        prompt_len + output_len for _, prompt_len, output_len in requests
+        prompt_len + output_len * args.num_sequences_to_sample for _, prompt_len, output_len in requests
```

Comment:
imo, we should not include tokens for re-computation in this kind of benchmarking to avoid the situation where throughput looks better than it should be because of re-computed tokens. If there are many re-computed tokens to generate valid output token, that is the problem of engine and the benchmark script should reflect this.

Reply:
I agree with @sunggg  here